torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -399,6 +399,10 @@ def batched_powerSGD_hook(state: PowerSGDState, bucket) -> torch.futures.Future:
     7) Computes M, which is approximately equal to PQ^T.
     8) Truncates the input tensor to the original length.
 
+    Note that this communication hook enforces vanilla allreduce for the first `state.start_powerSGD_iter` iterations.
```

Comment:
Do we have the default value for `state.start_powersgd_iter` documented in the docs for powerSGDState? Would be nice to ensure we have that, and maybe also specify the default start iteration here.